scripts/interop_start.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+"""
+2019/09/08 -- Interop start script.
+This script is intended for dockerfile deployment for interop testing.
+This script is fragile and subject to break as flags change.
+Use at your own risk!
+"""
+
+# Flags
+IDENTITY="" # P2P private key
+PEERS="" # Comma separated list of peers
+NUM_VALIDATORS="3" # Positive number of validators to operate.
+GEN_STATE="" # filepath to ssz encoded state.
+PORT="8000"
+
+# Constants
+BEACON_LOG_FILE="/tmp/beacon.log"
+VALIDATOR_LOG_FILE="/tmp/validator.log"
+YAML_KEY_FILE="/launch/keys.yaml"
+
+usage() {
+    echo "--identity=<identity>"
+    echo "--peer=<peer>"
+    echo "--num-validators=<number>"
+    echo "--gen-state=<file path>"
+    port "--port=<port number>"
+}
+
+while [ "$1" != "" ];
+do
+    PARAM=`echo $1 | awk -F= '{print $1}'`
+    VALUE=`echo $1 | sed 's/^[^=]*=//g'`
+
+    case $PARAM in
+        --identity)
+            IDENTITY=$VALUE
+            ;;
+        --peers)
+            PEERS+=" $VALUE"
```

Comment:
@prestonvanloon we had a space as a delim, does that work? If not, can you change the script to better suit Prysm?

Reply:
ah, OK i can change it

Reply:
Changed this line to concat with comma delim